os/drivers/testcase/Make.defs

```diff
@@ -20,7 +20,21 @@
 ifeq ($(CONFIG_KERNEL_TEST_DRV),y)
 
 CSRCS += kernel_testcase_drv.c
-CSRCS += test_task_init.c
+CSRCS += test_task.c
+CSRCS += test_sched.c
+CSRCS += test_clock.c
+CSRCS += test_timer.c
+CSRCS += test_sem.c
+
+ifeq ($(CONFIG_SCHED_HAVE_PARENT),y)
```

Comment:
If test_group.c needs both CONFIG_SCHED_HAVE_PARENT and CONFIG_SCHED_CHILD_STATUS, 
please change ifeq($(A)$(B),yy).

Reply:
Thanks.
I will change this.